http.rst

```diff
@@ -1,13 +1,80 @@
 HTTP
 ====
 
-A static variable representing a HTTP request.
+A HTTP request.
+
+Typical Usage
+-------------
+::
+
+  ForEach(originalRecords, originalRecord -> {
+      ForEach(responseRecords, record -> {
+        String description = "hello";
+        String originalMessageTimestampText = Convert.ToString(Convert.ToLocalTime(record.OriginalMessageTimestamp to String), "{0:MM/dd/yyyy H:mm:ss}");
+        HttpMessage httpMessage = Http.CreateMessage();
```

Comment:
let's just keep line 14-26

and also try to make it as generic as possible, so take away the process specific variables